docs/using/installing/index.rst

```diff
@@ -219,11 +229,11 @@ Using Rackspace
 Another way to get a Flocker cluster running is to use Rackspace.
 You'll probably want to setup at least two nodes.
 
-#. Create a new Cloud Server running Fedora 20
+#. Create a new Cloud Server
```

Comment:
Why is "Cloud Server" title case (and not "new"). I don't mind as long as we are consistent within and between titles.